openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/client/CubeOpenShiftConfiguration.java

```diff
@@ -334,4 +335,135 @@ public long getHttpClientTimeout() {
     public OpenShiftClient getClient() {
         return client;
     }
+
+    @Override
+    public String toString() {
+
+        String SEP = System.getProperty("line.separator");
+        StringBuilder content = new StringBuilder();
+
+        content.append("CubeOpenShiftConfiguration: ").append(SEP);
+
+        if (getNamespace() != null) {
```

Comment:
Can you remove all parameters that are already set in kubernetes config class, so the process of stringify common parameters are done in the same place.